public/js/app/components/AnalysisMenu.jsx

```diff
@@ -94,17 +94,9 @@ function AnalysisMenu(props) {
     ladokLoading,
   } = state
 
-  //* ****************************** SEMESTER DROPDOWN ******************************* */
-  //* ********************************************************************************* */
-  function toggleDropdown(event) {
-    event.preventDefault()
-    setState({
-      dropdownOpen: !dropdownOpen,
```

Comment:
Good that you removed the function `toggleDropdown` . To finish cleaning up of unused states, on line 47 and 83 there are leftovers of `dropdownOpen`, which should be cleaned up as well